### DISCUSSION TITLE
New Icons

### DISCUSSION THREAD

**Discussion by Jdochoa** [mariuz/flamerobin#161] [General]
Hi all. 
We need new icons for the new nodes. 

- DB Triggers
- DDL Triggers
- PSQL Functions
- Global Temporaries
- Indexes
- Packages
- Users

The files exist in de source code, but are by copy the others icons.

somebody can help?

**Comment by arvanus** [comment]
Also, we'll need a icon for 
- Run query  in background #125

**Comment by luronumen** [comment]
Hi @Jdochoa 
What is image file type that you need? png, ico?

**Reply by arvanus** [reply]
Flamerobin uses .xpm
You can look some samples at:
https://github.com/mariuz/flamerobin/tree/master/res

**Comment by luronumen** [comment]
Hi @Jdochoa  and @arvanus 

I have attached the icons used by the [RedExpert](https://github.com/red-soft-ru/redexpert/tree/master/guide/img):

- [DatabaseIcons.zip](https://github.com/mariuz/flamerobin/files/5873255/DatabaseIcons.zip)

They have a more modern look and perhaps this would be a cool opportunity to update all FlameRobin icons.

Best Regards,
Luciano

**Reply by mariuz** [reply]
OK License is GPL 3 , we need to mention in the readme https://github.com/red-soft-ru/redexpert/blob/master/LICENSE.txt (some of our icons are LGPL for example)
and we need to add the license in the res folder https://github.com/mariuz/flamerobin/tree/master/res

**Comment by Jdochoa** [comment]
thank you, i will work on this.

**Comment by luronumen** [comment]
Retest result on _FlameRobin 0.9.3.7_: **Partial Completed!**

The **Database**, **Domains**, **Exceptions** and **Roles** nodes are still using the old icons:

![Icons](https://user-images.githubusercontent.com/8019466/109941003-688c1a80-7cb1-11eb-849a-411fd110ce1f.png)

**Comment by luronumen** [comment]
I am attaching the missing icons (_Databases_, _Domains_, _Exceptions_, _Roles_, _Server_ and _System Domains_): [Icons.zip](https://github.com/mariuz/flamerobin/files/6082419/Icons.zip)

Best Regards,
Luciano

**Comment by luronumen** [comment]
I am attaching the missing Table Columns icons (Column, Foreign and Primary): [TableColumns.zip](https://github.com/mariuz/flamerobin/files/6092663/TableColumns.zip)

![Columns](https://user-images.githubusercontent.com/8019466/110158733-7e403380-7dc8-11eb-9af3-30dd057e6721.png)

Best Regards,
Luciano

**Comment by Jdochoa** [comment]
This was only the first of the changes. I'll keep working on it.

**Reply by luronumen** [reply]
> This was only the first of the changes. I'll keep working on it.

Thank @Jdochoa You are the guy :D

**Comment by luronumen** [comment]
Hi @Jdochoa 

To complete this discussion we just have to update the _Home_, _Domains_, _Exceptions_, _Roles_, _System Roles_ and _System Domains_ icons.
I am attaching the missing icons: [Icons.zip](https://github.com/mariuz/flamerobin/files/6526250/Icons.zip)

![FlameRobinIcons](https://user-images.githubusercontent.com/8019466/119223637-eb628b80-bad0-11eb-945f-1c73b9338923.png)

Best Regards,
Luciano

**Comment by luronumen** [comment]
Attaching the icon pack to FlameRobin: [Icons.zip](https://github.com/mariuz/flamerobin/files/6608301/Icons.zip)

![FlameRobin](https://user-images.githubusercontent.com/8019466/121006460-4ea61c00-c767-11eb-9f28-c123e606600e.PNG)
